mpp/include/mpp_util.inc

```diff
@@ -1091,7 +1093,9 @@ end function rarray_to_char
     integer,save :: i
     logical      :: l_open
 
-!  9 is reserved for etc_unit
+    ! Negative value for default/invalid unit number
+    get_unit = -1
```

Comment:
Again, I see this as a gfortran bug.  As above with stdlog get_unit will either return a number between 10 and 99 or throw a fatal error halting execution.  So I don't see how this is a case of trying to use something before it is defined.